pkg/nodeinfo/node_info.go

```diff
@@ -127,6 +129,8 @@ func (p *provider) GetNodePools(filters ...Filter) []NodePool {
 		}
 		nodePool.Kernel = kernel
 
+		nodePool.ContainerRuntime = getContainerRuntime(node)
```

Comment:
do you assume that container runtime is the same for all nodes that are part of the pool ?

technically every attribute that is not part of the key in nodePoolMap may be different and you just save the last node that match.

Reply:
ack.